sdk/storage/Azure.Storage.DataMovement/src/TransferJobInternal.cs

```diff
@@ -263,14 +263,9 @@ public void DisposeHandlers()
         public async Task TriggerJobCancellationAsync()
         {
             if (!_dataTransfer._state.CancellationTokenSource.IsCancellationRequested)
-            {
-                _dataTransfer._state.TriggerCancellation();
-            }
-
-            // Set CancellationInProgress if we are not already pausing
-            if (StorageTransferStatus.PauseInProgress != _dataTransfer.TransferStatus)
```

Comment:
Did this status message not matter is setting it to paused later on?

Reply:
This was here to make sure we didn't set it to cancelled if it was Pausing instead. Now that this has been moved into the check to see fi the token has been cancelled, we should not need the check and the pause operation will set PauseInProgress and then cancel the token so this code will not run.